landoapi/tasks.py

```diff
@@ -160,3 +160,20 @@ def admin_remove_phab_project(
         objectIdentifier=revision_phid,
         transactions=transactions,
     )
+
+
+@celery.task(
+    autoretry_for=(IOError, PhabricatorCommunicationException),
+    default_retry_delay=20,
+    max_retries=3 * 20,  # 20 minutes
```

Comment:
`3 * 20` seems like a large number of max retries? (i.e. 60). The comment is not very clear as to what 20 minutes is referring to (maybe it's meant for `default_retry_delay`?)